pyatv/__init__.py

```diff
@@ -27,8 +33,13 @@ async def scan(
     identifier: Optional[Union[str, Set[str]]] = None,
     protocol: Optional[Union[Protocol, Set[Protocol]]] = None,
     hosts: List[str] = None,
+    async_zeroconf: Optional[AsyncZeroconf] = None,
```

Comment:
```suggestion
    zeroconf: Optional[AsyncZeroconf] = None,
```

I like it short (type hint tells us it's async).

Reply:
Does aiozc work here?

since there is a zeroconf object inside the async version it makes it confusing to name it the same

Reply:
Yeah, that is fine!

Reply:
ea6f728